test/slack/web/documentation_test.exs

```diff
@@ -48,8 +83,8 @@ defmodule Slack.Web.DocumentationTest do
 
       module_functions = Slack.Web.Oauth.V2.__info__(:functions)
 
-      assert {:access, 3} in module_functions
-      assert {:access, 4} in module_functions
+      assert {:access, 0} in module_functions
+      assert {:access, 1} in module_functions
```

Comment:
ℹ️  The `oauth.v2.access.json` has been updated (according to the API docs) that non of these params are required anymore.